### PR TITLE
Fix/one silver

### PR DESCRIPTION
**Pull request by RUsami**
Fixing double silver and Pillagers dropping silver

### CODE REVIEW

**Comment by GildedToxin** [comment]
will merge after mega dump merge
also waiting on manifest from mega dump to make the manifest for this one

**Comment by Jonah-Hansen** [comment]
closes #288

**Review by Jonah-Hansen — APPROVED** [review]

**Pull request merged**
